tests failed

```

======================================================================
ERROR: test_trigger (custom.enikshay.integrations.ninetyninedots.tests.test_repeaters.TestRegisterPatientRepeater)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 258, in inner
    return helper(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 231, in __call__
    call_with_settings(fn_with_pre_and_post, run_config.settings, args, kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 246, in call_with_settings
    fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 226, in fn_with_pre_and_post
    run_config.pre_run(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/form_processor/tests/utils.py", line 166, in <lambda>
    pre_run=lambda *args, **kwargs: args[0].setUp(),
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 258, in inner
    return helper(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 231, in __call__
    call_with_settings(fn_with_pre_and_post, run_config.settings, args, kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 246, in call_with_settings
    fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 226, in fn_with_pre_and_post
    run_config.pre_run(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/form_processor/tests/utils.py", line 166, in <lambda>
    pre_run=lambda *args, **kwargs: args[0].setUp(),
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 258, in inner
    return helper(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 231, in __call__
    call_with_settings(fn_with_pre_and_post, run_config.settings, args, kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 246, in call_with_settings
    fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 226, in fn_with_pre_and_post
    run_config.pre_run(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/form_processor/tests/utils.py", line 166, in <lambda>
    pre_run=lambda *args, **kwargs: args[0].setUp(),
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 258, in inner
....
RuntimeError: maximum recursion depth exceeded
```

--- a/custom/enikshay/integrations/ninetyninedots/tests/test_repeaters.py
+++ b/custom/enikshay/integrations/ninetyninedots/tests/test_repeaters.py
@@ -73,7 +73,6 @@ class ENikshayRepeaterTestBase(ENikshayCaseStructureMixin, TestCase):
 
 class TestRegisterPatientRepeater(ENikshayRepeaterTestBase):
 
-    @run_with_all_backends
     def setUp(self):
         super(TestRegisterPatientRepeater, self).setUp()
 

